modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb

```diff
@@ -22,7 +22,8 @@ def perform(power_of_attorney_id)
     rescue Errno::ENOENT
       rescue_file_not_found(power_of_attorney)
     rescue ClaimsApi::StampSignatureError => e
-      power_of_attorney.update(signature_errors: e.detail)
+      signature_errors = (power_of_attorney.signature_errors || []).push(e.detail)
```

Comment:
in the case that `power_of_attorney.signature_errors` is already defined, wouldn't we still want to add push `e.detail` and not just in the latter case where it's getting added to an empty array?

Reply:
I'm not sure I understand the question - the code it supposed to take the current `signature_errors` and add the current error to it. the `|| []` bit should make it an empty array if it's `nil`, I presume it will never be `nil` but figure it couldn't hurt to put that bit in just in case.

Reply:
oh my bad, I missed the parentheses. I was reading it as
```ruby 
signature_errors = power_of_attorney.signature_errors || [].push(e.detail)
```
your logic makes total sense, disregard my question.